test/e2e/run_test.go

```diff
@@ -493,7 +493,9 @@ USER bin`
 				Skip("Kernel does not support blkio.weight")
 			}
 		}
-
+		if podmanTest.Host.Distribution == "ubuntu" {
```

Comment:
This is fragile, and would be better expressed as a (TBD) check against `/sys/block/sda/queue/scheduler`, but I'll settle for a FIXME comment referencing the BZ (sorry, I couldn't find the BZ in `setup_environment.sh`).

Reply:
I actually considered this, but it only turns out slightly less fragile: Who's to say the host has a SCSI block device?  Could be virtio, ide, nvme, usb, etc.  

I think the proper fix, is to (somehow) examine the set of available io-schedulers in the kernel...which I'm not eager to attempt (and do it badly).  I think it's better to throw in a short-term kludge and rely on someone more golang-savvy to solve properly later.

Reply:
(I'm still working on opening issues)

Reply:
...in fact, it looks like _almost_ the correct skip is already here: `if _, err := os.Stat("/sys/fs/cgroup/blkio/blkio.weight"); os.IsNotExist(err) {`.  However, in the case of Ubuntu 20.10 w/ CGroupsV2, it doesn't cause the test to be skipped.  More, deeper debugging is necessary.  I'm working on opening an issue now.